packages/vaex-viz/vaex/viz/mpl.py

```diff
@@ -632,7 +633,10 @@ def _selection_name(name):
                     if facets > 1:
                         colorbar.ax.set_title(label)
                     else:
-                        colorbar.ax.set_ylabel(label)
+                        if colorbar_label is not None:
+                            colorbar.ax.set_ylabel(colorbar_label)
+                        else:
+                            colorbar.ax.set_ylabel(label)
```

Comment:
this can be cleaned up by doing: colorbar.ax.set_ylabel(colorbar_label or label) and the rest as well.